src/main/java/com/uci/utils/service/UserService.java

```diff
@@ -392,17 +391,15 @@ public static User getUserByFullName(String fullName, String campaignName) throw
 	}
 
 	public static User getInfoForUser(String userID) {
-		FusionAuthClient staticClient = new FusionAuthClient("c0VY85LRCYnsk64xrjdXNVFFJ3ziTJ91r08Cm0Pcjbc",
-				"http://134.209.150.161:9011");
+		FusionAuthClient staticClient = getFusionAuthClient();
```

Comment:
Fusion Auth Client url & token is still visible in file, Please change it to use from the application properties & env variables.